Tests/SonarScanner.MSBuild.PreProcessor.Test/TargetsInstallerTests.cs

```diff
@@ -90,7 +90,6 @@ public void InstallTargetsFile_Overwrite()
             var msBuildPathSettings = new MsBuildPathSettings();
 
             InstallTargetsFileAndAssert(sourceTargetsContent1, expectCopy: true);
-            msBuildPathSettings.GetImportBeforePaths().Should().HaveCount(7, "Expecting six destination directories");
```

Comment:
I had to remove the assertion since it's not always valid. Depending on the machine configuration, the scanner can add copy the target file in more than 6 directories.

e.g.
```
2024-02-09T15:10:56.0361009Z  DEBUG: Installed SonarQube.Integration.ImportBefore.targets to C:\Windows\system32\config\systemprofile\AppData\Local\Microsoft\MSBuild\4.0\Microsoft.Common.targets\ImportBefore
2024-02-09T15:10:56.0361432Z  DEBUG: Installed SonarQube.Integration.ImportBefore.targets to C:\Windows\system32\config\systemprofile\AppData\Local\Microsoft\MSBuild\10.0\Microsoft.Common.targets\ImportBefore
2024-02-09T15:10:56.0361850Z  DEBUG: Installed SonarQube.Integration.ImportBefore.targets to C:\Windows\system32\config\systemprofile\AppData\Local\Microsoft\MSBuild\11.0\Microsoft.Common.targets\ImportBefore
2024-02-09T15:10:56.0362270Z  DEBUG: Installed SonarQube.Integration.ImportBefore.targets to C:\Windows\system32\config\systemprofile\AppData\Local\Microsoft\MSBuild\12.0\Microsoft.Common.targets\ImportBefore
2024-02-09T15:10:56.0362680Z  DEBUG: Installed SonarQube.Integration.ImportBefore.targets to C:\Windows\system32\config\systemprofile\AppData\Local\Microsoft\MSBuild\14.0\Microsoft.Common.targets\ImportBefore
2024-02-09T15:10:56.0363080Z  DEBUG: Installed SonarQube.Integration.ImportBefore.targets to C:\Windows\system32\config\systemprofile\AppData\Local\Microsoft\MSBuild\15.0\Microsoft.Common.targets\ImportBefore
2024-02-09T15:10:56.0363546Z  DEBUG: Installed SonarQube.Integration.ImportBefore.targets to C:\Windows\system32\config\systemprofile\AppData\Local\Microsoft\MSBuild\Current\Microsoft.Common.targets\ImportBefore
2024-02-09T15:10:56.0363977Z  DEBUG: Installed SonarQube.Integration.ImportBefore.targets to C:\Windows\SysWOW64\config\systemprofile\AppData\Local\Microsoft\MSBuild\4.0\Microsoft.Common.targets\ImportBefore
2024-02-09T15:10:56.0364373Z  DEBUG: Installed SonarQube.Integration.ImportBefore.targets to C:\Windows\SysWOW64\config\systemprofile\AppData\Local\Microsoft\MSBuild\10.0\Microsoft.Common.targets\ImportBefore
2024-02-09T15:10:56.0364765Z  DEBUG: Installed SonarQube.Integration.ImportBefore.targets to C:\Windows\SysWOW64\config\systemprofile\AppData\Local\Microsoft\MSBuild\11.0\Microsoft.Common.targets\ImportBefore
2024-02-09T15:10:56.0365163Z  DEBUG: Installed SonarQube.Integration.ImportBefore.targets to C:\Windows\SysWOW64\config\systemprofile\AppData\Local\Microsoft\MSBuild\12.0\Microsoft.Common.targets\ImportBefore
2024-02-09T15:10:56.0365558Z  DEBUG: Installed SonarQube.Integration.ImportBefore.targets to C:\Windows\SysWOW64\config\systemprofile\AppData\Local\Microsoft\MSBuild\14.0\Microsoft.Common.targets\ImportBefore
2024-02-09T15:10:56.0365942Z  DEBUG: Installed SonarQube.Integration.ImportBefore.targets to C:\Windows\SysWOW64\config\systemprofile\AppData\Local\Microsoft\MSBuild\15.0\Microsoft.Common.targets\ImportBefore
2024-02-09T15:10:56.0366342Z  DEBUG: Installed SonarQube.Integration.ImportBefore.targets to C:\Windows\SysWOW64\config\systemprofile\AppData\Local\Microsoft\MSBuild\Current\Microsoft.Common.targets\ImportBefore
2024-02-09T15:10:56.0366819Z  DEBUG: Installed SonarQube.Integration.targets to C:\sonar-ci\_work\1\s\TestResults\Deploy_SYSTEM 20240209T151053\45\sonarqube\bin\targets
```